pkg/kubelet/config/config_test.go

```diff
@@ -413,3 +415,36 @@ func TestPodUpdateLabels(t *testing.T) {
 	expectPodUpdate(t, ch, CreatePodUpdate(kubetypes.UPDATE, TestSource, pod))
 
 }
+
+func TestPodRestore(t *testing.T) {
+	// Write checkpoint file
```

Comment:
i suggest to use library function checkpoint/WritePod to write checkpoint, it helps to avoid hard code pod spec in string.
```
CreateValidPod()
WritePod()
```
although you need to initialize the checkpointmanager, this checkpointmanager should not affect restore.